src/content/docs/iast/introduction.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: Introduction to IAST
+tags:
+    - New Relic IAST
+    - IAST
+    - Exploitable vulnerabilities
+metaDescription: Learn what New Relic IAST is and how you can use it.
+redirects:
+    - /docs/iast/use-iast
+    - /docs/iast/overview
+freshnessValidatedDate: never
+---
+
+The security of your software applications is essential to protect sensitive data, maintain the trust of your users, and comply with regulatory requirements. New Relic Interactive Application Security Testing (IAST) can help you to prevent cyberattacks and breaches on your applications by probing your running code for exploitable vulnerabilities.
+
+New Relic IAST helps address some of the limitations of more traditional application security tools like Static Application Security Testing (SAST), Dynamic Application Security Testing (DAST), and penetration testing that struggle to keep pace with modern application delivery, especially applications based on based on distributed microservices. IAST helps bridge the gap, saving you money and time, as it's substantially less likely to generate false positive findings, which increase developer confidence and drive efficiency. You get double duty from your manual or automated QA tests to continuously detect and prioritize vulnerabilities that are exploitable, along with guidance on how to eliminate the risk.
```

Comment:
This opening paragraph is pretty long and complicated. The first sentence is a few lines long. Can we work together to break this down into smaller chunks?